source/composition/structuredefinition-Composition.xml

```diff
@@ -455,6 +515,76 @@
         <map value="DocumentReference.author"/>
       </mapping>
     </element>
+    <element id="Composition.participant">
+      <path value="Composition.participant"/>
+      <short value="Identifies supporting entities, including parents, relatives, caregivers, insurance policyholders, guarantors, and others related in some way to the patient"/>
+      <definition value="Identifies supporting entities, including parents, relatives, caregivers, insurance policyholders, guarantors, and others related in some way to the patient. A supporting person or organization is an individual or an organization with a relationship to the patient. A supporting person who is playing multiple roles would be recorded in multiple participants (e.g., emergency contact and next-of-kin)."/>
+      <min value="0"/>
+      <max value="*"/>
+      <type>
+        <code value="BackboneElement"/>
+      </type>
+    </element>
+    <element id="Composition.participant.type">
+      <path value="Composition.participant.type"/>
+      <short value="AUT | AUTHEN | CST | LA | RCT | SBJ"/>
+      <definition value="Meaning and purpose of participation, in creation of the clinical document. The list includes: data entry person, informant, primary information recipient, and tracker. Other types are allowed."/>
```

Comment:
So..... I know we have the comment about the excluded types; and that's why we're using the ClinicalDocument value set.... 

But I thought AUTHEN and LA were handled by attester. This seems like a bug with that value set....